fragdenstaat_de/fds_cms/templates/foirequest/cms_plugins/map.html

```diff
@@ -0,0 +1,311 @@
+{% load i18n %}
+{% load static %}
+{% load leaflet_tags %}
+{% leaflet_js %}
+{% leaflet_css %}
+<div class="foirequest-map-container">
+    <div id="map" style="height: 600px;"></div>
+</div>
+<div id="map-legend" class="foirequest-legend" style="display: none;">
+    <b>{% trans "Status" %}</b>
+    <br>
+    <i style="background:#ffc107"></i> {% trans "Pending" %}
```

Comment:
Wondering if we should have these colours centrally in froide codebase or use bootstrap classes.